src/person/controllers/update_person_controller.py

```diff
@@ -80,6 +82,20 @@ def process(self):
             self._person.prompt_questions.set(prompt_questions)
             modify_attribute(self._person, "prompt_answers", json.dumps(prompt_answers))
 
+        print(fcm_registration_token)
+        print(self._person.fcm_registration_token)
+        if (
+            fcm_registration_token is not None
+            and self._person.fcm_registration_token != fcm_registration_token
+        ):
+            self._person.fcm_registration_token = fcm_registration_token
+            fcm_device = GCMDevice.objects.create(
```

Comment:
How long before the fcm_registration token expires so that the person's FCM registration token isn't the one? I'm trying to understand why we create a new device token if the registration token isn't the same. Is the intention that once the user registers Device 1 and then registers Device 2, we should make their device Device 2? But then when I switch back to Device 1, I create another device (without having deleted the previous one) and now there are two Device 1s?

Reply:
i feel u, i'll update it so that old devices are deleted when new ones are created